lib/danger/danger_core/plugins/dangerfile_danger_plugin.rb

```diff
@@ -85,10 +85,10 @@ def import_dangerfile(opts)
         elsif opts.key?(:gem)
           import_dangerfile_from_gem(opts[:gem])
         else
-          raise "`import` requires a Hash with either :github or :gem"
+          raise "`import` requires a Hash with either :github, :gem, :gitlab_project_id, or :path"
```

Comment:
```suggestion
          raise "`import` requires a Hash with either :github, :gitlab, :gitlab_project_id, :gem or :path"
```

Reply:
Fixed this message, and used `:gitlab` instead of `:gitlab_project_id` since that is now fixed. Thanks!